docs/adrs/000-adr-template.md

```diff
@@ -0,0 +1,22 @@
+# Title
```

Comment:
The adr tools https://github.com/npryce/adr-tools suggest a template that is a tiny bit differen from this template.

```
# 1. Record architecture decisions

Date: 2023-04-26

## Status

Accepted

## Context

We need to record the architectural decisions made on this project.

## Decision

We will use Architecture Decision Records, as [described by Michael Nygard](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).

## Consequences

See Michael Nygard's article, linked above. For a lightweight ADR toolset, see Nat Pryce's [adr-tools](https://github.com/npryce/adr-tools).
```
That would allow us to use the tool to create adrs.
I haven't checked whether the template that is used by the tool can be altered, though.

Reply:
I'm not sure whether I'd be willing to install an additional binary to create a markdown file. :thinking: 

It would probably be a neater way to provide a file template for IDEA.

Reply:
To use that tool is not necessary, but possible then.

A convenient way to create adrs in IDEA would be nice as well.

Reply:
Is there a convenient way to share file templates in IDEA?